test/geometry/test_homography.py

```diff
@@ -392,12 +394,15 @@ def test_smoke(self, device, dtype):
 
     @pytest.mark.parametrize("batch_size, num_points", [(1, 4), (2, 5), (3, 6)])
     def test_shape(self, batch_size, num_points, device, dtype):
-        B, N = batch_size, num_points
-        points1 = torch.rand(B, N, 2, device=device, dtype=dtype)
-        points2 = torch.rand(B, N, 2, device=device, dtype=dtype)
-        weights = torch.ones(B, N, device=device, dtype=dtype)
-        H = find_homography_dlt_iterated(points1, points2, weights, 5)
-        assert H.shape == (B, 3, 3)
+        if sys.platform == 'darwin' and torch.__version__ == '1.9.1':
+            pytest.skip("Known bug in torch 1.9.1 on macos")
```

Comment:
i suggest to use the following pattern
https://github.com/kornia/kornia/blob/e432b7b2e13eb48602fd361947ef9119988920a3/test/tracking/test_planar_tracking.py#L34

torch.__version__ not always gives a constant return pattern

Reply:
I have rewritten the code like this as I want it to only skip if it's mac. Is that okay?
if sys.platform == 'darwin' and torch_version_le(1, 9, 1):
            pytest.skip("Known bug in torch 1.9.1 on macos")